### ISSUE TITLE
one method for updating object attributes

### ISSUE THREAD

**Issue by tkoomzaaskz** [jashkenas/underscore#1869]
I miss an underscore method that would take an object and update all it's attributes, within a iteratee and preserver all keys. Say I've got `{a:1, b:2, c:3}` and I want to use a square function to receive: `{a:1, b:4, c:9}`. Current solution is to do something like this:

```js
_.object(_.map(original, function (value, key) {
    return [key, value * value];
}));
```

but I don;t like that. It's not that readable - you need to use another 2 functions and know that _.object constructs another object from pairs. There could be one simple function for this, since updating all object attributes is not a rare task. How about adding it to underscore? An example solution can be found here:
http://stackoverflow.com/questions/26013802/underscore-js-iterate-over-object-attributes-and-modify-them

**Comment by greenify** [comment]
> fully agree with @megawac. This is a common case afterall. (@tkoomzaaskz)

Any chance to get this into 1.8 e.g. with #1953?

**Comment by megawac** [comment]
@jashkenas can you make a decision on this and #1953?

**Comment by jashkenas** [comment]
This is a dupe of the implemented #1953, no? Let's move over there.
